types/types.go

```diff
@@ -204,7 +204,6 @@ func HashFromHex(h string, pad Pad) (Hash, error) {
 }
 
 // HashFromHexPtr parses a hash in hex format and returns a *Hash type.
-// If hash is longer than 32 bytes, it returns an error.
 // It returns nil if the hash is invalid.
 func HashFromHexPtr(h string, pad Pad) *Hash {
```

Comment:
actually this name `HashPtrFromHex` fits this function better. Don't you think so?